lib/WebService/Stripe.pm

```diff
@@ -70,6 +70,10 @@ method get_balance(:$headers) {
     return $self->get( "/v1/balance", {}, headers => $headers );
 }
 
+method get_balance_history(:$headers) {
```

Comment:
I don't think the underlying route path needs to match the name. The route path returns balance_transaction objects. So 'get_balance_transaction' and 'get_balance_transactions' matches what you get. Also, what's the singular of 'get_balance_history'? :)